api-reference/endpoint/get-software.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: "Get Software Info"
```

Comment:
We have more than just `Software` in our db. We have operating systems, devices, services, and packages. Should use entity here